internal/speakerlist/speakerlist.go

```diff
@@ -74,7 +75,7 @@ func New(logger gokitlog.Logger, nodeName, bindAddr, bindPort, secret, namespace
 	loggerout := gokitlog.NewStdlibAdapter(gokitlog.With(sl.l, "component", "Memberlist"))
 	mconfig.Logger = golog.New(loggerout, "", golog.Lshortfile)
```

Comment:
Those 2 lines generate logs like:
```
{"caller":"net.go:210","component":"Memberlist","msg":"[DEBUG] memberlist: Stream connection from=10.10.52.152:51718","ts":"2021-07-07T20:33:58.686178124Z"}
{"caller":"net.go:785","component":"Memberlist","msg":"[DEBUG] memberlist: Initiating push/pull sync with: 10.10.52.153:7946","ts":"2021-07-07T20:34:26.625477039Z"}
{"caller":"net.go:210","component":"Memberlist","msg":"[DEBUG] memberlist: Stream connection from=10.10.52.152:52002","ts":"2021-07-07T20:34:28.689981448Z"}
{"caller":"net.go:210","component":"Memberlist","msg":"[DEBUG] memberlist: Stream connection from=10.10.52.153:38596","ts":"2021-07-07T20:34:45.650096602Z"}
{"caller":"net.go:785","component":"Memberlist","msg":"[DEBUG] memberlist: Initiating push/pull sync with: 10.10.52.153:7946","ts":"2021-07-07T20:34:56.628673276Z"}
```

Is it possible from `[DEBUG]` to set the level to debug ?

Reply:
https://github.com/hashicorp/memberlist/search?q=logger.Printf

Reply:
It was possible with some c/p from the `NewStdlibAdapter` and adapting it to extract the level from the logs submitted by memberlist. Please have a look to the last commit.

Reply:
Thanks !